### PR TITLE
W-13480276 promoting-api-instance-kl.adoc

### PR DESCRIPTION
**Pull request by kevinzlu-salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/promote-api-task.adoc [hunk 1]
```diff
@@ -3,23 +3,20 @@ ifndef::env-site,env-github[]
...
-. Select or deselect entities to include in the promotion: Policies, SLAs, Alerts, and API Configuration.
-. Optionally, if your external client provider differs in the environment to which your are promoting the API, select the provider in the *Client provider* list.
-. Click *Promote*.
+You can only promote an entity (Policies, Alerts, and more) if you have both permission to view the source and to manage the destination over each entity. Typical view permissions include `View API Configuration` and `View Policies`, as explained in xref:environment-permission-task.adoc[To Grant Permission to Access Environments]. 
+
+Set the permissions as needed before proceeding to the following steps:
+
+include::partial$task-promote-api.adoc[]
 
 Promoting an API affects only the API definition, not the existing deployed applications subscribed to the API. After promoting an API, the API Name remains the same, but the API ID changes.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/promote-api-task.adoc]:
```suggestion
Promoting an API instance affects only the API definition, not the existing deployed applications subscribed to the instance. After promoting an API, the *API Label* remains the same, but the *API Instance ID* changes.
```

File: modules/ROOT/pages/promote-api-task.adoc [hunk 2]
```diff
@@ -3,23 +3,20 @@ ifndef::env-site,env-github[]
...
 
-For existing applications that need to subscribe to a promoted API, you need to make the following changes:
+For existing applications that need to subscribe to a promoted API:
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/promote-api-task.adoc]:
```suggestion
For existing applications that must subscribe to a promoted API instance:
```

File: modules/ROOT/pages/promote-api-task.adoc [hunk 3]
```diff
@@ -3,23 +3,20 @@ ifndef::env-site,env-github[]
...
 
 . Update the API ID.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/promote-api-task.adoc]:
```suggestion
. Update the *API Instance ID* within the application.
```

File: modules/ROOT/pages/promote-api-task.adoc [hunk 4]
```diff
@@ -3,23 +3,20 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-A tight integration of Anypoint Platform components extends the use of environments to API management. In this procedure, you change the environment of an API.
+Before promoting an API instance to another environment:
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/promote-api-task.adoc]: See google doc:

https://docs.google.com/document/d/1_CNwMZhAFZGxBLBNo2VRuKZnmPGAy2-ikd-P5YUrgVE/edit?usp=sharing

**Review by glenn-rodgers-sf — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/promote-api-task.adoc [hunk 5]
```diff
@@ -3,25 +3,19 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-A tight integration of Anypoint Platform components extends the use of environments to API management. In this procedure, you change the environment of an API.
+Before promoting an API instance to another environment ensure that you have the `Manage APIs Configuration` permission.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/promote-api-task.adoc]:
```suggestion
Before promoting an API instance to another environment, ensure that you have the `Manage APIs Configuration` permission.
```

**Review by bjoson-mulesoft — DISMISSED** [review]

**Review by juanpjacob — APPROVED** [review]

File: modules/ROOT/pages/promote-api-task.adoc [hunk 6]
```diff
@@ -3,25 +3,19 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-A tight integration of Anypoint Platform components extends the use of environments to API management. In this procedure, you change the environment of an API.
+Before promoting an API instance to another environment, ensure that you have the `Manage APIs Configuration` permission.
```
Comment (hannanelson) [inline modules/ROOT/pages/promote-api-task.adoc:6]: I don't think we change fonts for permissions.

Comment (hannanelson) [reply modules/ROOT/pages/promote-api-task.adoc:6]: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-fontchoices

File: modules/ROOT/pages/promote-api-task.adoc [hunk 7]
```diff
@@ -3,25 +3,19 @@ ifndef::env-site,env-github[]
...
 
-You must have `Manage APIs Configuration` permission to promote an API and `Access Management` permission to set permissions for viewing and managing an entity. You can only promote an entity (Policies, Alerts, and so forth) if you have permission to view the source and permission to manage the destination over each entity. Typical view permissions are `View API Configuration` and `View Policies`, as explained in xref:environment-permission-task.adoc[To Grant Permission to Access Environments]. Set the permissions as needed before proceeding.
+You can only promote an entity, such as policies or alerts, if you have both the permissions to view the entity and to manage its destination. For example, you must have the `View Policies` and `Manage Policies` permissions to promote a policy.
```
Comment (hannanelson) [inline modules/ROOT/pages/promote-api-task.adoc:8]: font

File: modules/ROOT/pages/promote-api-task.adoc [hunk 8]
```diff
@@ -3,25 +3,19 @@ ifndef::env-site,env-github[]
...
+include::partial$task-promote-api.adoc[]
 
-For existing applications that need to subscribe to a promoted API, you need to make the following changes:
+Promoting an API instance affects only the API definition, not the existing deployed applications subscribed to the instance. After promoting an API, the *API Label* remains the same, but the *API Instance ID* changes.
 
-. Update the API ID.
+For existing applications that must subscribe to a promoted API instance:
+
+. Update the *API Instance ID* within the application.
 . Because each environment has a different client ID and secret, update the application's client ID and secret.
```
Comment (hannanelson) [inline modules/ROOT/pages/promote-api-task.adoc:19]:
```suggestion
. Update the application's client ID and secret.
+
Each environment has a different client ID and secret.
```

**Review by hannanelson — APPROVED** [review]

**Review by glenn-rodgers-sf — APPROVED** [review]

**Pull request merged**